mappings/net/minecraft/entity/ai/brain/sensor/SecondaryJobSiteSensor.mapping

```diff
@@ -0,0 +1 @@
+CLASS apm net/minecraft/entity/ai/brain/sensor/SecondaryJobSiteSensor
```

Comment:
Why was this renamed? Look at the ID in `SensorType`:
```java
SECONDARY_POIS = SensorType.<SecondaryPointsOfInterestSensor>register("secondary_pois", SecondaryPointsOfInterestSensor::new);
```

Reply:
i saw it was using secondary job site from memory module types